#! /bin/sh /usr/share/dpatch/dpatch-run
## 17-fix_tail_syntax_in_xversion.h.dpatch by Sven Mueller <debian@incase.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: fix tail syntax in xversion.h (use "tail -n 1" instead of "tail -1")

@DPATCH@
--- a/imap/xversion.sh
+++ b/imap/xversion.sh
@@ -22,4 +22,4 @@ find .. -name '*.[chly]' -print | \
 	match ($0, pattern) {
 	    printf "\"%s\"\n", substr($0, RSTART, RLENGTH)
 	}' pattern="$DATEPAT $TIMEPAT" | \
-	sort | tail -1 >> xversion.h
+	sort | tail -n 1 >> xversion.h
